__tests__/endpoint.test.js

```diff
@@ -81,14 +81,67 @@ describe("/api/articles/", () => {
       });
   });
 });
+describe("/api/articles/:article_id/comments", () => {
+  test("The endpoint will respond with an array of comments from the given article ID", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        body.comments.forEach((comment) => {
+          expect(typeof comment.comment_id).toBe("number");
+          expect(typeof comment.votes).toBe("number");
+          expect(typeof comment.created_at).toBe("string");
+          expect(typeof comment.author).toBe("string");
+          expect(typeof comment.body).toBe("string");
+          expect(typeof comment.article_id).toBe("number");
+        });
+      });
+  });
+
+  test("The endpoint will respond with array", () => {
+    return request(app)
+      .get("/api/articles/9/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(Array.isArray(body.comments)).toBe(true);
+      });
+  });
+
+  test("404 given a bad article ID the endpoint will respond with an error", () => {
+    return request(app)
+      .get("/api/articles/555667/comments")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("not found");
+      });
+  });
+  test("500 given an article ID which is not a number the endpoint will respond with an error", () => {
```

Comment:
We would expect an invalid article_id to return a 400 error as it is a bad request. Please refactor this before merging to main.